doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -209,71 +210,139 @@ infrastructure itself from the workloads compute resources.
+networking solution as a whole, may be delivered by the **"Default CNI Plugin"**,
+or the **"CNI multiplexer/meta-plugin"** if it is deployed. For example:
+    - The integration of SDN solutions required by `req.inf.ntw.05` is enabled
+via CNI integration.
+    - IP Address Management (**IPAM**) of the various networks can be provided
+by one or more IPAM plugins, which can be part of a CNI plugin, or some other
+component (i.e. external SDN solution) - it is key that there are no overlapping
+IP addresses within a cluster, and if multiple IPAM solutions are used that
+they are co-ordinated in some way (as required by `req.inf.ntw.10`).
+- **Service Mesh**: The well known service meshes are "application service meshes"
+that address and interact with the application layer 7 protocols (eg.: HTTP)
+only. Therefore, their support is not required in this architecture, as these
+service meshes are outside the scope of the infrastructure layer of this
+architecture.
+
+<p align="center"><img src="../figures/ch03_networking.png" alt="Kubernetes Networking Architecture" Title="Kubernetes Networking Architecture" width="100%"/></p>
+<p align="center"><b>Figure 3-2:</b> Kubernetes Networking Architecture</p>
+
+<!--The above diagram is maintained here:
+https://wiki.lfnetworking.org/display/LN/CNTT+RA2+-+Kubernetes+-+Diagrams+-+Networking-->
+
+There are several types of low latency and high throughput networks required by
 telco workloads: signalling traffic workloads and user plane traffic workloads.
 Networks used for signalling traffic are more demanding than what a standard
 overlay network can handle, but still do not need the use of user space
 networking. Due to the nature of the signalling protocols used, these type of
-networks require NAT-less communication documented in infra.net.cfg.003 and will
+networks require NAT-less communication documented in `infra.net.cfg.003` and will
 need to be served by a CNI plugin with IPVLAN or MACVLAN support. On the other
 hand, the low latency, high throughput networks used for handling the user plane
 traffic require the capability to use a user space networking technology.
 
+| Requirement | Networking Solution with Multus | Networking Solution with DANM | Networking Solution with Tungsten Fabric | Networking Solution with NSM |
+|---|---|---|---|---|
+| The overlay network encapsulation protocol needs to enable ECMP in the underlay (`infra.net.cfg.002`) | Supported via the additional CNI plugin | Supported via the additional CNI plugin | Supported | TBC |
+| NAT (`infra.net.cfg.003`) | Supported via the additional CNI plugin | Supported | Supported | TBC |
+| Network Policies (Security Groups) (`infra.net.cfg.004`) | Not supported | Not supported <sub>(1)<sub> | Supported | Supported via a CNI Network Plugin that supports Network Policies |
+| SFC support (`infra.net.cfg.005`) | Not relevant | Not relevant | Not relevant | Not relevant |
```

Comment:
Why include, if "not relevant"? 
Is there anything in a solution that may be more beneficial or may be negatively impacting to SFCs

Reply:
Not my area of expertise, but I had always thought the SFC thing was managed at the orchestration layer, rather than within a Kubernetes cluster? I don't know of any "gotchas" regarding SFCs - i.e. if a CNF can run, can it be used in a chain? @rgstori what do you think?

Reply:
agree, SFC is more suited to 3GPP NSMF/NSSMF and ETSI MANO components. although the implementation of these _can be done e.g. with k8s operators_, it's not the point of CNTT RA2-compliant platform to "support SFC" (eg ONAP being a better place for modelling/maintaining/operating SFCs). I'm ok with removing the line entirely, or leaving it as is if it's needed as explicit memento of how SFC doesn't strictly belong here

Reply:
Thanks - I have removed the line in this table, and have created an issue to review/remove the requirement #2189